kv/txn_coord_sender.go

```diff
@@ -758,28 +756,26 @@ func (tc *TxnCoordSender) heartbeat(id string) {
 					Reply: reply,
 				}
 
-				task := tc.stopper.StartTask()
-				if !task.Ok() {
+				if !tc.stopper.RunTask(func() {
```

Comment:
I'm not familiar with this code to judge. I'd rather confine this PR to a purely structural one as much as possible. I can look at removing the need for the continue in another PR